Index: BusinessLayer/Middlepoint/UserMiddlepoint.cs
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/BusinessLayer/Middlepoint/UserMiddlepoint.cs b/BusinessLayer/Middlepoint/UserMiddlepoint.cs
new file mode 100644
--- /dev/null	(date 1637416090730)
+++ b/BusinessLayer/Middlepoint/UserMiddlepoint.cs	(date 1637416090730)
@@ -0,0 +1,7 @@
+namespace BusinessLayer.Middlepoint
+{
+    public class UserMiddlepoint
+    {
+        
+    }
+}
\ No newline at end of file
